### PR TITLE
Connected controller and service methods

### PR DESCRIPTION
**Pull request by antoninguerre**

### CODE REVIEW

**Comment by antoninguerre** [comment]
closes #30

File: PostalRateCalculator-Backend/src/main/java/PostalRateCalculator/service/EnvelopeService.java [hunk 1]
```diff
@@ -65,24 +65,24 @@ private static void weightRangeCheck(double weight, WeightUnit weightUnit) {
...
+//        if (sizeUnitString.equals("in")) { sizeUnit = SizeUnit.Inches; }
+//
+//        if (weightUnitString.equals("g")) { weightUnit = WeightUnit.Grams; }
+//        if (weightUnitString.equals("oz")) { weightUnit = WeightUnit.Ounces; }
+//
+//        Envelope envelope = new Envelope(width, length, weight, sizeUnit, weightUnit);
+//        envelope = envelopeRepository.save(envelope);
+//
+//        return envelope;
+//    }
```
Comment (JulietteDebray) [inline PostalRateCalculator-Backend/src/main/java/PostalRateCalculator/service/EnvelopeService.java:87]: so... nothing is happening here?

**Review by JulietteDebray — APPROVED** [review]
not quite sure what you did here but looks okay!

File: PostalRateCalculator-Backend/src/main/java/PostalRateCalculator/service/EnvelopeService.java (hunk 1 above)
Comment (antoninguerre) [reply PostalRateCalculator-Backend/src/main/java/PostalRateCalculator/service/EnvelopeService.java:87]: that's gonna be removed at the end
I just used that to test that i could add envelopes to the database

**Pull request merged**